napari/view_layers.py

```diff
@@ -544,18 +576,6 @@ def view_shapes(
         {'opaque', 'translucent', and 'additive'}.
     visible : bool
         Whether the layer visual is currently being displayed.
-    title : string
```

Comment:
Why are these doc strings being cut? I think they appear on every `view_*` method, and args are still there

Reply:
Sorry, that was an accident! Fixed.